### DISCUSSION TITLE
Suggestion about the setup

### DISCUSSION THREAD

**Discussion by mzfr** [pry0cc/axiom#239] [Q&A] [answered]
I haven't tried axiom before so wanted to try set it up but I have a question, where do you recommend setting this up? Locally on my system or on a VPS?

I am asking this because I see that `axiom-fleet` runs like N numbers of other droplets to do all the work that is to be done right? so is it really necessary to run these fleets from another VPS? Is there any performance issue that you noticed/found when we run `axiom-fleet` or `axiom-exec` locally from our system as compared to when the same is done through VPS?

**Comment by 0xtavian** [comment, marked as answer]
@mzfr Great question. and thank you for opening a discussion! There is a certainly a performance and reliability benefit with running axiom controller from a VPS. SSH connections are less likely to drop, which at the moment is a big one. RSYNC and all operations, except actual scanning, should feel faster than running locally. However, you certainly can run locally, especially with the new axiom-scan. You'd just have to worry if your electricity/internet ever stops during a scan as that would more than likely cause the ssh sockets to fail. Same if you computer goes to sleep. Whatever solution you chose, run axiom in a tmux or screen session.

**Reply by mzfr** [reply]
@0xtavian Thanks for replying. Yeah I guess I'll try it using both and will try to see which one suits me best :)
